src/main/java/marquez/api/models/DatasourceResponse.java

```diff
@@ -0,0 +1,21 @@
+package marquez.api.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class DatasourceResponse {
+
+  @JsonProperty("name")
+  String name;
```

Comment:
Is there a reason we are defaulting instances variables to package-private ?

Reply:
No specific reason. Happy to make them private. In that case, we'd also want to go back and make sure that's the case in other places, such as `NamespaceResponse`.

Reply:
Great. We'll want to have issues first before we start address other classes

Reply:
https://jira.we.co/browse/DATAP-979
Also a good first issue.

Reply:
For this particular class, it'll be handled in a subsequent PR.

Reply:
Please see PRs #394, #387, and #384, where this is fixed.